internal/fs/fs.go

```diff
@@ -53,7 +63,8 @@ func HasFilepathPrefix(path, prefix string) bool {
 		return false
 	}
 
-	var d, p string
+	d := vnPath
```

Comment:
Given that we know that Windows will _always_ treat volume name as case insensitive, this seems unnecessary - why isn't it sufficient to just test the `strings.ToLower()` volume names, as you do above?